v2/constants.go

```diff
@@ -52,7 +52,7 @@ const (
 	ApiDeviceByProfileNameRoute   = ApiDeviceRoute + "/" + Profile + "/" + Name + "/{" + Name + "}"
 	ApiDeviceByServiceIdRoute     = ApiDeviceRoute + "/" + Service + "/" + Id + "/{" + Id + "}"
 	ApiDeviceByServiceNameRoute   = ApiDeviceRoute + "/" + Service + "/" + Name + "/{" + Name + "}"
-	ApiDeviceNameCommandNameRoute = ApiDeviceRoute + "/" + Name + "/{" + DeviceName + "}" + "/" + Command + "/{" + CommandName + "}"
```

Comment:
please also remove the CommandName constant

Reply:
https://github.com/edgexfoundry/go-mod-core-contracts/pull/460/commits/673266bae183075f6fd13f200aa979784cf454e1 removes commandName constant.